backend/src/test/java/pl/edu/agh/apdvbackend/fakes/GroupFakes.java

```diff
@@ -23,18 +23,22 @@ public static Group getGroup() {
     }
 
     @Builder
-    public static Group buildNewGroup(
+    private static Group buildNewGroup(
             Long id,
             String name,
             Set<User> usersInGroup,
             List<GroupEndpoint> groupEndpoints
     ) {
         final var group = new Group();
         group.setId(id);
-        group.setName(name);
-        group.setUsersInGroup(usersInGroup);
-        group.setGroupEndpoints(groupEndpoints);
+        group.setName(getOrElse(name, "dummy group name"));
+        group.setUsersInGroup(getOrElse(usersInGroup, Collections.emptySet()));
+        group.setGroupEndpoints(getOrElse(groupEndpoints, Collections.emptyList()));
 
         return group;
     }
+
+    private static <T> T getOrElse(T value, T defaultValue) {
```

Comment:
mamy optionala w domu, optional w domu:

Reply:
W sumie to nie pomyślałem, czy `@Builder` ogarnia `Optional'e` jako argumenty metod, ale może sprawdzę w przyszłych PR'kach

Reply:
No idea, tbh nie jest to jakoś specjalnie bolące w oczy - albo ja mam swoje standardy